scripts/o3de/ExportScripts/export_source_built_project.py

```diff
@@ -279,14 +283,14 @@ def export_standalone_parse_args(o3de_context: exp.O3DEScriptExportContext, expo
                                               parser=parser,
                                               key=exp.SETTINGS_SEED_LIST_PATHS.key,
                                               dest='seedlist_paths',
-                                              description='Path to a seed list file for asset bundling. Specify multiple times for each seed list.',
+                                              description='Path to a seed list file for asset bundling. Specify multiple times for each seed list. You can also specify wildcard patterns.',
```

Comment:
Does `validate_project_artifact_paths` actually handle wildcard paths? I dont think I see anywhere that the seedlist_paths undergo the same preprocessing as seedfile_paths